express/scripts/templates.js

```diff
@@ -353,6 +353,14 @@ async function updateBlocks(data) {
   }
 }
 
+const redirects = await fetch('/redirects.json?limit=99999').then((resp) => resp.json());
```

Comment:
I'd rewrite to first check that the request was successful before calling resp.json(). Otherwise if the request fails then you get a critical javascript error and no other javascript will be executed, which gives the user a confusing white screen
`
const redirects = await fetch('/redirects.json?limit=99999');
if (redirects.ok) {
  const json = await redirects.json();
  const toRedirect = json.data.find((row) => row.Source === window.location.pathname);
  if (toRedirect) {
    window.location.assign(toRedirect.Destination);
  }
}
`